roles/jboss-scan/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+
+---
+# Scan linux systems for JBoss EAP or Wildfly Installations
+    - name: Gather jboss.jar-ver
+      raw: FOUND=""; for jar in `find {{scan_dirs}} -xdev -name 'jboss-modules.jar' 2>/dev/null | grep -v '\.installation/patches'`; do VERSION=$(java -jar ${jar} -version 2> /dev/null | grep version | sed 's/.*version\s//g'); inode=$(stat -c '%i' "${jar}"); fs=$(df  -T "${jar}" | grep "/dev" | sed 's/ .*//'); ctime=$(stat ${jar} | grep 'Change' | grep -oP '[1-2][0-9]{3}-[0-1][0-9]-[0-3][0-9]'); if [ ! -z "${VERSION}" ]; then if [ ! -z "$FOUND" ]; then FOUND="$FOUND; $VERSION**$ctime"; else FOUND=${VERSION}'**'${ctime}; fi; fi; done; echo ${FOUND}
+      register: jboss.jar-ver
```

Comment:
wouldn't mind updating the fact names either to be EAP focus, but that could be for another PR.

Reply:
Will do.